### PR TITLE
Review SonarCloud Issues

### LINKED ISSUE #1019: Review SonarCloud Issues
### Problem

The SonarCloud report for the SDK needs to be reviewed.

### Solution

Review and address findings in report.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by easpaas**
**Description**:

This PR modifies several files in order to satisfy the SonarCloud issues.

**Related issue(s)**:

Fixes #1019 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by easpaas** [comment]
Most SonarCloud issues were ignored. This PR includes the few exceptions.

File: src/util.js [hunk 1]
```diff
@@ -37,11 +37,7 @@ export const FUNCTION_CONVERT_TO_NUMBER_PARSE_ERROR =
  * @returns {boolean}
  */
 export function isNonNull(variable) {
-    if (variable == null || variable == undefined) {
-        return false;
-    } else {
-        return true;
-    }
+    return variable == null || variable == undefined ? false : true;
```
Comment (danielakhterov) [inline src/util.js:40]: Pretty sure you can simply this into `return variable != null`

**Review by danielakhterov — CHANGES_REQUESTED** [review]

File: src/util.js (hunk 1 above)
Comment (easpaas) [reply src/util.js:40]: @danielakhterov great catch!

Comment (easpaas) [reply src/util.js:40]: If an empty string can be considered false-y then we can just reduce this to `!!variable`.

**Review by danielakhterov — APPROVED** [review]

**Pull request merged**